### ISSUE TITLE
Matrix `eigs` eigenvectors are out of order compared to eigenvalues

### ISSUE THREAD

**Issue by Lazersmoke** [josdejong/mathjs#1844]
The [documentation example](https://mathjs.org/docs/reference/functions/eigs.html) has this line:

```javascript
const ans = math.eigs(H) // returns {values: [E1,E2...sorted], vectors: [v1,v2.... corresponding vectors]}
```
Which tells us mathematically that:
```
H * v1 = E1 * v1
```
However, this doesn't actually work:
```javascript
>>> H = [[1,0,0],[0,1,0],[0,0,-1]]; e =math.eigs(H)
>>> math.multiply(H,e.vectors[0]) 
[0,1,0]
>>> math.multiply(e.values[0],e.vectors[0])
[-0,-1,-0]
```
These should give the same result! In this particular case, `e.vectors[1]` goes with `e.values[0]`:
```javascript
>>> math.multiply(H,e.vectors[1])
[0,0,-1]
>>> math.multiply(e.values[0],e.vectors[1])
[-0,-0,-1]
```

The reason is that the eigenvalues are sorted, but the eigenvectors are still in the original, unsorted order, and thus not "corresponding" as implied by the documentation.

Slow workaround:
```javascript
var e = math.eigs(H)
// Sort in place
e.vectors.sort((v,w) => {
  // Compare by eigenvalues
  return math.dot(math.multiply(H,v),v) - math.dot(math.multiply(H,w),w)
})
```

**Comment by phtown** [comment]
Possibly the issue here is one of documentation. The columns of e.vectors are the eigenvectors in the corresponding order to e.values:
`math.multiply(H, math.column(e.vectors, 0)) === math.multiply(e.values[0], math.column(e.vectors, 0))`

**Comment by josdejong** [comment]
Docs are fixed now in v7.0.1
